vt/client.py

```diff
@@ -597,6 +598,9 @@ async def scan_file_async(self, file, wait_for_completion=False):
     # AppEngine's upload handler doesn't like the filename*=UTF-8''foobar field
     # and fails with this Content-Disposition header.
 
+    if not isinstance(file, io.IOBase):
```

Comment:
nit: I would move the type check before the comment, as the comment refers to the logic related to the payload handling and is not related to the file check.